raiwidgets/raiwidgets/responsibleai_dashboard_input.py

```diff
@@ -445,9 +445,6 @@ def change(x):
             predictions = convert_to_list(
                 self._analysis.model.forecast(filtered_data_df),
                 EXP_VIZ_ERR_MSG)
-            # forecast should return a flat list of predictions
-            if all([len(p) == 1 for p in predictions]):
```

Comment:
Because the predictions returned by the automl model are already flattened so it tries to access len(p) where p is a float. I think this code was added for some other type of model Roman was using.